pipeline/08-api.R

```diff
@@ -88,13 +79,15 @@ for (town in towns) {
       meta_pin, meta_card_num, meta_class, pred_card_initial_fmv,
       api_prediction, api_prediction_rounded,
       all_of(top_predictors),
-      meta_modeling_group, meta_tieback_proration_rate,
       starts_with("char_"),
+      ccao_is_corner_lot,
       starts_with("loc_"),
       starts_with("time"),
       starts_with("prox_"),
       starts_with("acs5_"),
-      starts_with("other_")
+      starts_with("other_"),
+      meta_sale_count_past_n_years, ccao_is_active_exe_homeowner,
+      ccao_n_years_exe_homeowner,
```

Comment:
Grouping these in with the `Other` predictors seemed most appropriate, but let me know if you think there's a better group for them in the Card sheet.